runtime/src/lib.rs

```diff
@@ -270,6 +270,30 @@ impl staking::Trait for Runtime {
     type SessionInterface = Self;
     type Time = Timestamp;
     type RewardCurve = RewardCurve;
+    type AddOrigin = collective::EnsureProportionMoreThan<_2, _3, AccountId, GovernanceCollective>;
+    type RemoveOrigin =
+        collective::EnsureProportionMoreThan<_2, _3, AccountId, GovernanceCollective>;
+    type ComplianceOrigin =
```

Comment:
Compliance should still be done via an identity check (e.g. KYB) (which can be stubbed) rather than the governance committee.

Governance committee is the first stage, compliance checks is the ongoing check each ERA that the validator(s) still has a valid KYB check.

The actual identity check can be stubbed out for now.

Reply:
See comments below.